messages/grid/grid.ar.yml

```
@@ -191,3 +191,6 @@ kendo:
 
     # The label of the checkbox column select all checkbox
     selectAllCheckboxLabel: Select All Rows
+
+    # The label of the Grid pager
+    pagerLabel: 'Pager page {currentPage} of {totalPages}'
```

```suggestion
    pagerLabel: 'Page {currentPage} of {totalPages}'
```

Pager seems redundant in the message.

---

> Pager seems redundant in the message.

After further discussion and having in mind that the Pager component role is "navigation", and Google use "Page navigation" in their pagination section, we decided to go with "Page navigation, page {currentPage} of {totalPages}".